packages/shared-lib/src/peripheralDevice/peripheralDeviceAPI.ts

```diff
@@ -72,6 +72,7 @@ export enum PeripheralDeviceCategory {
 	PLAYOUT = 'playout',
 	MEDIA_MANAGER = 'media_manager',
 	PACKAGE_MANAGER = 'package_manager',
+	API = 'api',
```

Comment:
I wonder if this should be API or LIVE_STATUS? 
Do we expect/want other gateway types to exist under this category without needing any changes to support them in meteor?

Reply:
I've renamed this to `LIVE_STATUS`, given that e.g. package manager has its own category I'm happy to have a special category until we find any commonality.